run

```diff
@@ -118,6 +118,9 @@ function runstuff {
         bash -c "dotnet test /app/SIL.AppBuilder.Portal.Backend.Tests/"
     ;;
 
+    ci)
```

Comment:
I added it so that I could run ci stop during testing.  I left it because it seemed it might be useful in the future and I didn't think having it in the run script would present a problem.